google/cloud/security/common/data_access/sql_queries/create_tables.py

```diff
@@ -73,3 +73,12 @@
         PRIMARY KEY (`id`)
     ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 """
+
+CREATE_GSUITE_ACCOUNT_GROUPS_TABLE = """
+    CREATE TABLE `{0}` (
+        `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,
```

Comment:
Looking at the groups resource definition, the "alias" might be interesting:
https://developers.google.com/admin-sdk/directory/v1/reference/groups#resource

Do you think that we would need aliases?  Could aliases be used somehow in permissioning? If so, it would help to collect it.